### DISCUSSION TITLE
Improvements in podcasts tab

### DISCUSSION THREAD

**Discussion by daniloribeiro00** [ciderapp/Cider#1143] [Feature Request]
Hey! Thanks for the excellent app!
It would be nice if it was possible to sort the followed podcasts by name and last updated.
Also, it would be great if the playback arrow could change to be like the Apple Podcasts app for going back in 15s and forward in 30s.
Finally, another suggestion is bring a recently updated podcasts tab with the option to mark the podcast as played so it would also update in the iPhone app, even though I don’t know if their API allows this feature.
Thanks.

**Comment by owainharris** [comment]
Yes, this would be great!
